devel/diffstat/Portfile

```diff
@@ -8,9 +8,10 @@ description             Make histogram from diff-output
 long_description        diffstat reads the output of diff and displays a histogram\
                         of the insertions, deletions, and modifications per-file.\
                         It is useful for reviewing large, complex patch files.
-homepage                http://invisible-island.net/diffstat/
+homepage                https://invisible-island.net/diffstat/
 platforms               darwin
-master_sites            ftp://ftp.invisible-island.net/diffstat/
+master_sites            ftp://ftp.invisible-island.net/diffstat/ \
+                        https://invisible-island.net/datafiles/release/diffstat.tar.gz
```

Comment:
Thanks for the review. The filename in the diffstat url was a mistake, I was copying the full url then failed to zap the filename as I did for the others.

Looking at the build bot's logs, I realize now that it doesn't try fetching from the new url as long as the version is cached on the distfiles server (just like a regular macports install would), so I should've checked for myself earlier that just because both the https and ftp have the unversioned file didn't mean that the https would have versioned ones like ftp does. (Maybe it's so that there's one less place for the developer to update a url when the version changes.) I will change these to use invisible-mirror.net/archives/ .